rascal-vscode-extension/README.md

```diff
@@ -27,8 +27,9 @@ The Rascal VScode extension currently is bundled with the following libraries:
 
 ## Requirements
 
-* Installed JDK _8_ (11 is also allowed)
-* JDK on PATH *or* configured in JAVA_HOME environment variable
+* Installed JDK _11_ 
+* JAVA_HOME set to this JDK
```

Comment:
The Java detector is not that strict, if it finds java on the `PATH` it's also fine.

Reply:
fixed